bodhi/server/templates/fragments.html

```diff
@@ -128,7 +128,7 @@
           </div>
       </div>
       </div>
-      
+
```

Comment:
These look like unrelated white space fixes to me, not sure what the consensus is how we want to deal with these kind of fixes. @cverna @pypingou @abompard?